R/lasso-deployment.R

```diff
@@ -25,6 +23,28 @@
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
 #' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use. Defaults to 2.
```

Comment:
I think it's ok if there's a discrepancy there. The default is 2, but examples happen to use 1.